### PR TITLE
add docstring to functions

### LINKED ISSUE #12: minor fixes
- Doesn't match function name. Also, use snake_case for file names: https://github.com/sct-pipeline/csa-atrophy/blob/f4d95b0cf25d21e2f59b7d4b0078951405d45dea/csa-fetch-dataset.sh#L6
- For all your code, use docstrings for describing your functions, so we can create automatic documentation with sphinx:
https://github.com/sct-pipeline/csa-atrophy/blame/f4d95b0cf25d21e2f59b7d4b0078951405d45dea/affine_rescale.py#L50

### PR DESCRIPTION
**Pull request by PaulBautin**
For all csa-atrophy code, use docstrings for describing functions, so we can create automatic documentation with sphinx.

Fixes #12

### CODE REVIEW

File: affine_rescale.py [hunk 1]
```diff
@@ -48,6 +48,12 @@ def get_parser():
 #MAIN
 ############################################################
 def main(fname, coef_r):
+    """
+    Main function, rescales input images according to coef_r
+    :param fname: input image
+    :param coef_r: rescaling coefficient
+    :return: rescaled image
```
Comment (jcohenadad) [inline affine_rescale.py:55]: The function does not return anything. You can write in the description that it writes stuff in the disk, but the function itself does not return any variable

**Comment by PaulBautin** [comment]
This branch can be closed because "Docstring" has been added to old functions. "Docstring" will continue to be implemented in master on new files and functions.

**Comment by PaulBautin** [comment]
@jcohenadad, I think it is best to close this PR. Last commit simply merged master back into this PR branch "fixes".

**Comment by jcohenadad** [comment]
in the future pls keep PR changes isolated across branches-- otherwise it makes the review process confusing

**Comment by jcohenadad** [comment]
> @jcohenadad, I think it is best to close this PR.

@PaulBautin i though you wanted to close this????? not merge it!

**Comment by PaulBautin** [comment]
@jcohenadad, i have restored branch but everything has already been merged (no diff between branches fixes and master). However fixes keeps memory of what has been merged too soon.

**Pull request merged**